pkgs/applications/audio/yasr/30_conf

```diff
@@ -0,0 +1,22 @@
+Index: yasr/yasr.conf
+===================================================================
+--- yasr.orig/yasr.conf	2008-01-31 14:01:50.000000000 +0100
++++ yasr/yasr.conf	2012-01-28 21:43:09.027710410 +0100
+@@ -55,13 +55,13 @@ cursor tracking=full
+ #synthesizer=doubletalk
+ #synthesizer=bns
+ #synthesizer=apollo
+-synthesizer=emacspeak server
+-#synthesizer=speech dispatcher
++#synthesizer=emacspeak server
++synthesizer=speech dispatcher
```

Comment:
Yes, you're right about 'shell'. To be honest I just took the patch from the debian yasr package. I need to figure out what synthesizer would be preferred in nixos (at least, eflite and freetts do not seem to work out of box).